src/raft/raft.rs

```diff
@@ -101,6 +105,9 @@ pub struct Config {
     /// quorum is not active for an electionTimeout.
     pub check_quorum: bool,
 
+    /// read_only_option specifies how the read only request is processed.
```

Comment:
The comments here in etcd/raft are just duplication of the comments for the definition ReadOnlyOption. It's better to remove these duplication, rather than just copy them around.